VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It is registration specific. It is formatted to look well on the registrations screen (in others words, it can't be easily abstracted to spinners in general...at least not now)